src/components/text/__tests__/index.driver.spec.tsx

```diff
@@ -44,3 +45,45 @@ describe('Text', () => {
     });
   });
 });
+
+// jest.mock('react-native/Libraries/ReactNative/I18nManager', () => ({
```

Comment:
@NitzanWix 
Instead of mocking RN it might be simpler to override/mock our Constants.isRLT value
See here
https://github.com/wix/react-native-ui-lib/blob/master/src/components/sortableGridList/__tests__/usePresenter.spec.tsx

Reply:
He's testing the style here, so I don't think that'll work.
Although I'm not sure this is the best way to test this bug, an alternative might be with screenshots and I think we don't want to rely on that here.

Reply:
I'm trying to test the following. overriding/mocking in Constants will work for RTL but I want to also mock the platform. Currently this doesn't work and pops a invalid hook usage error.
https://github.com/wix/react-native-ui-lib/blob/068b4d00174bbc13f1ef33dfcb811d9ab7dee369/src/components/text/index.tsx#L195-L203